### DISCUSSION TITLE
Any plan to support python 3.8?

### DISCUSSION THREAD

**Discussion by alqurri77** [microsoft/tensorflow-directml#364] [Ideas] [answered]
Hi;

I'm using python 3.8 and I wonder if you are planning to support python 3.8 any time soon?

Thank you!

**Comment by PatriceVignola** [comment, marked as answer]
We don't have plans to support 3.8 for tensorflow-directml 1.15 since the official tensorflow 1.15 release doesn't support it either, but stay tuned for DirectML on TF2 which will definitely support python 3.8!

**Reply by alqurri77** [reply]
Thank you ... any expected date for DirectML on TF2 ?

**Reply by PatriceVignola** [reply]
We don't have a definitive date yet, but we'll be able to share more soon.

**Reply by alqurri77** [reply]
Thanks again ... I'm using Keras/tensorflow 2.7 with Python 3.8. Would directml works, as well as Keras/tensorflow 2.7 if I downgrade to Python 3.7
